net/yggdrasil/Makefile

```diff
@@ -1,16 +1,16 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=yggdrasil
-PKG_VERSION:=0.3.5
-PKG_RELEASE:=4
+PKG_VERSION:=0.3.6
+PKG_RELEASE:=1
 
 PKG_SOURCE_URL:=https://codeload.github.com/yggdrasil-network/yggdrasil-go/tar.gz/v$(PKG_VERSION)?
```

Comment:
Swap the order of PKG_SOURCE and PKG_SOURCE_URL

Reply:
its now matching same order as contributing.md